org.eu.wabarc.wayback.yml

```diff
@@ -0,0 +1,32 @@
+app-id: org.eu.wabarc.wayback
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: wayback
+finish-args:
+  - --share=network
+  - --filesystem=xdg-documents
+modules:
+  - name: wayback
+    buildsystem: simple
+    build-commands:
+      - |
+        set -eu pipefail
+        version="0.18.1"
+        tar -xvf wayback-linux-*-${version}.tar.gz
+        rm -f wayback-linux-*-${version}.tar.gz
+        mv wayback-linux-* wayback
+      - install -Dm755 -t /app/bin wayback
+    sources:
+      - type: file
+        url: https://github.com/wabarc/wayback/releases/download/v0.18.1/wayback-linux-amd64-0.18.1.tar.gz
+        sha256: bca11c1ca33cb03f631efa4d0b304852826ace00580963699520d32e52e7add9
+        only-arches:
+          - x86_64
```

Comment:
this doesn't do what you think it does.

there still will be (an attempt) to build an aarch64 flatpak.